workflows/hifiasm/tbev.wdl

```diff
@@ -0,0 +1,36 @@
+version 1.0
+
+#tbev.wdl: to run triobin/trioeval and test memory size, coded by Charlie Bi
```

Comment:
@cbi-star Is this file just for testing? If so can we talk about re-organizing testing wdl's vs actual workflow wdls?

Reply:
Several workflows were added for testing. But the workflows don't have to be used for testing exclusively. For instance, users with yak output and assemblies could just run `triobin` using this workflow easily.
On the other hand, there are many workflows and it's overwhelming. In addition to improve the documentation, I agree with you and re-organizing into an intuitive structure would be great. Do you have any suggestions how we should reorganize?